examples/index.html

```diff
@@ -224,7 +221,18 @@ <h2>Improve this tool</h2>
       var sequencer;
     })
   </script>
-
+  <script src="uppy.js"></script>
+    <script>
+      var uppy = Uppy.Core()
+        .use(Uppy.Dashboard, {
+          inline: true,
+          target: '#dropzone'
+        })
+        .use(Uppy.Tus, {endpoint: 'https://master.tus.io/files/'})
+       uppy.on('complete', (result) => {
+        console.log('Upload complete! We’ve uploaded these files:', result.successful)
```

Comment:
Ah, we should remove this log line, i think. And is this where the `use()` for the webcam access would go too?

Reply:
Yeah.

Reply:
Also, just wondering, did this properly add the files to the sequencer for the next step?